go.mod

```diff
@@ -36,6 +35,8 @@ require (
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.6.1
+	github.com/vishvananda/netlink v1.1.0 // indirect
+	github.com/vishvananda/netns v0.0.0-20201230012202-c4f3ca719c73 // indirect
```

Comment:
Seem to be unrelated

Reply:
I did `go mod tidy` so I haven't touched it

Reply:
Weirdly `go mod tidy` actually adds these here